python/pfs/drp/stella/SpectrumSetContinued.py

```diff
@@ -6,7 +6,7 @@
 from lsst.pipe.base import Struct
 import lsst.afw.image as afwImage
 
-from pfs.datamodel.drp import PfsArm
+from pfs.drp.datamodel import PfsArm
```

Comment:
This illustrates the confusion with the naming of the drp_stella-specific classes: it's not obvious that with this change the drp_stella subclasses of the datamodel are being used.